app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,23 @@
+class AccountActivationsController < ApplicationController
+  before_action :find_user, only: :edit
+
+  def edit
+    if !@user.activated? && @user.authenticated?(:activation, params[:id])
+      @user.activate
+      log_in @user
+      flash[:success] = t("account.activate_success")
+      redirect_to @user
+    else
+      flash[:danger] = t("account.activate_fail")
+      redirect_to root_url
+    end
+  end
+end
+
+def find_user
+  @user = User.find_by email: params[:email]
+  return if @user
+
+  flash[:danger] = t("users_controller.not_found")
```

Comment:
```suggestion
  flash[:danger] = t "users_controller.not_found"
```
check lại convention nhé